client/me/purchases/payment/edit-card-details/index.jsx

```diff
@@ -53,12 +54,43 @@ const EditCardDetails = React.createClass( {
 		'postalCode'
 	],
 
+	componentWillReceiveProps( nextProps ) {
+		// Updates the form once with the stored credit card data as soon as they are available
+		if ( nextProps.card && ( ! this.props.card || ( nextProps.card.id !== this.props.card.id ) ) ) {
+			this.setState( {
+				form: formState.initializeFields( this.state.form, this.mergeCard( nextProps.card ) )
+			} );
+		}
+	},
+
+	/**
+	 * Merges the specified card object returned by the StoredCards store into a new object with only properties that
+	 * should be used to prefill the credit card form, and with keys matching the corresponding field names.
+	 *
+	 * @param card
+	 * @param fields
+	 */
+	mergeCard( card, fields: {} ) {
```

Comment:
Was this intended to be `fields = {}` to provide the default argument? This line broke after dropping [Flow](https://flowtype.org/) plugin in the Babel 6 upgrade (#1515) and is assumed to be corrected in 4b06d6ed57a01d794bef379cc8aae78474b03d21.